<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsWidget</class>
 <widget class="QWidget" name="ResultsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>594</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ResultsWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="vbox_results">
     <item>
      <widget class="QGroupBox" name="group_payrollMeta">
       <property name="title">
        <string>Payroll Summary</string>
       </property>
       <layout class="QGridLayout" name="grid_payrollMeta">
        <item row="0" column="0">
         <widget class="QLabel" name="label_payPeriod">
          <property name="text">
           <string>Payroll Period:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="value_payPeriod">
          <property name="text">
           <string>—</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_dateProcessed">
          <property name="text">
           <string>Date Processed:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="value_dateProcessed">
          <property name="text">
           <string>—</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group_resultsTable">
       <property name="title">
        <string>Employee Payroll Results</string>
       </property>
       <layout class="QVBoxLayout" name="vbox_resultsTable">
        <item>
         <widget class="QTableWidget" name="table_results">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>15</number>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>140</number>
          </attribute>
          <column>
           <property name="text">
            <string>Emp ID.</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Department</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Basic Salary</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Allowance</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>OT Pay</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Adjustments</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Gross Income</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>SSS Premium</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>PHIC Premium</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>HDMF Premium</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Loans</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>W-hold Tax</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Total Deduct</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Net Pay</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
